### DISCUSSION TITLE
Directly typing-in park admission price

### DISCUSSION THREAD

**Discussion by luotiansha** [OpenRCT2/OpenRCT2#21898] [Feature requests]
Making park fee can be typed in, just like fees for rides. So that we can type in 999 easily (cuz why not).

![屏幕快照 2024-04-26 的 02 54 21 上午](https://github.com/OpenRCT2/OpenRCT2/assets/77926812/7bee1639-d34f-490d-b28a-77c917bfb6ed)

**Comment by Harry-Hopkinson** [comment]
This was implemented in https://github.com/OpenRCT2/OpenRCT2/pull/21734 - if you use the latest develop build this feature will be implemented.

**Reply by luotiansha** [reply]
nice
